src/bqlfn.py

```diff
@@ -280,6 +281,35 @@ def bql_column_value_probability(bdb, generator_id, modelno, colno, value,
     r = metamodel.logpdf_joint(
         bdb, generator_id, targets, constraints, modelno)
     return ieee_exp(r)
+
+# XXX This is silly.  We should return log densities, not densities.
```

Comment:
There was some historical discussion in the past of what to return, i.e.
https://github.com/probcomp/bayeslite/issues/114